torch/csrc/autograd/python_cpp_function_attr.cpp

```diff
@@ -0,0 +1,54 @@
+#include "python_cpp_function_attr.h"
+
+#include "torch/csrc/autograd/python_variable.h"
+#include "torch/csrc/autograd/python_hook.h"
+
+namespace torch { namespace autograd {
+
+namespace attributes {
+
+PyObject* next_functions(THPCppFunction* self, PyObject* hook)
```

Comment:
I think it's better to keep these in `python_cpp_function.cpp` and try to not add more files.